docs/source/future/core/nrunner.rst

```diff
@@ -278,57 +278,52 @@ It's possible to interact with the runner features by using the
 command line.  This interface is not stable at all, and may be changed
 or removed in the future.
 
-You can run a "noop" runner with::
-
-  python3 -m avocado.core.nrunner runnable-run -k noop
-
-You can run an "exec" runner with::
-
-  python3 -m avocado.core.nrunner runnable-run -k exec -u /bin/uname --args='-a'
-
-You can run an "exec-test" runner with::
-
-  python3 -m avocado.core.nrunner runnable-run -k exec-test -u /bin/true
-
-You can run a "python-unittest" runner with::
-
-  python3 -m avocado.core.nrunner runnable-run -k python-unittest -u unittest.TestCase
-
-Trying it out - Avocado Plugins
--------------------------------
-
-Simple Avocado plugins for the runner features are also available.
-
 Runnables from parameters
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
 You can run a "noop" runner with::
 
-  avocado runnable-run -k noop
+  avocado-runner runnable-run -k noop
 
 You can run an "exec" runner with::
 
-  avocado runnable-run -k exec -u /bin/sleep -a 3
+  avocado-runner runnable-run -k exec -u /bin/uname --args='-a'
 
 You can run an "exec-test" runner with::
 
-  avocado runnable-run -k exec-test -u /bin/true
+  avocado-runner runnable-run -k exec-test -u /bin/true
 
 You can run a "python-unittest" runner with::
 
-  avocado runnable-run python-unittest unittest.TestCase
+  avocado-runner runnable-run -k python-unittest -u unittest.TestCase
 
 Runnables from recipes
 ~~~~~~~~~~~~~~~~~~~~~~
 
 You can run a "noop" recipe with::
 
-  avocado runnable-run-recipe examples/recipes/runnables/noop.json
+  avocado-runner runnable-run-recipe examples/nrunner/recipes/runnables/noop.json
 
 You can run an "exec" runner with::
 
-  avocado runnable-run-recipe examples/recipes/runnables/exec_sleep_3.json
+  avocado-runner runnable-run-recipe examples/nrunner/recipes/runnables/exec_sleep_3.json
 
 You can run a "python-unittest" runner with::
 
-  avocado runnable-run-recipe examples/recipes/runnables/python_unittest.json
+  avocado-runner runnable-run-recipe examples/nrunner/recipes/runnables/python_unittest.json
+
+Writing new runner scripts
```

Comment:
Maybe we could split this document thinking on our user cases.  But looks like this section is more appropriate on "Contributors Guide".

Not sure if we need this separation right now.

Reply:
I agree that the separation is needed, but it may be more confusing to split it now. Once we remove the *experimental* status of the nrunner, we will move all the related documentation away from this location anyway.